frontend/js/helpers/main.js

```diff
@@ -149,9 +153,14 @@ async function makeAPICall(path, values=null) {
             }
         }
     }  else {
-        var options = { method: 'GET' }
+        var options = { method: 'GET', headers: {} }
     }
 
+    let authentication_token = localStorage.getItem('authentication_token');
+    if(authentication_token == null) authentication_token = 'DEFAULT';
```

Comment:
I would not set a token default value as this might confuse people when debugging. As as far as I understand the backend it defaults to `DEFAULT` if there is no header.

Reply:
also resolved. See other comment